backend/src/friend/friend.controller.ts

```diff
@@ -43,4 +46,99 @@ export class FriendController {
   async getSentRequests(@GetUser() user: User) {
     return { sentRequests: user.friendRequestsSent };
   }
+
+  /*****************************************************************************/
+  // POST
+  /*****************************************************************************/
+  // Send request
+  @Post('sendFriendRequest')
+  async handleSendFriendRequest(
+    @GetUser() user: User,
+    @Body() payload: { userEmail: string },
```

Comment:
Maybe add a dto that shows up in the Swagger and validates input, to be on the safe side. Should be fast and easy, since it is already implemented.